applications/Unity.GrantManager/src/Unity.GrantManager.Web/wwwroot/colorsPalette.json

```diff
@@ -0,0 +1,3 @@
+{
+    "colors": ["#F8BA47", "#3470B1", "#7E5D21", "#A5792B"]
```

Comment:
Don't think we need to extract this to another file, hardcoding into the colors should be fine, I don't see us deploying just the colorsPalette on its own.